lib/macos.h

```diff
@@ -25,7 +25,7 @@
  ***************************************************************************/
 #include "curl_setup.h"
 
-#if defined(__APPLE__) && (!defined(TARGET_OS_OSX) || TARGET_OS_OSX)
+#if defined(__APPLE__) && TARGET_OS_MAC && !(defined(TARGET_OS_IPHONE) && TARGET_OS_IPHONE)
```

Comment:
Apparently you have line length restrictions:

```
./macos.h:28:92: warning: Longer than 79 columns (LONGLINE)
 #if defined(__APPLE__) && TARGET_OS_MAC && !(defined(TARGET_OS_IPHONE) && TARGET_OS_IPHONE)
```

What alternate formatting does this project prefer to address this?

Reply:
There are hundreds of examples present in the lib/ directory alone. Keep lines narrower than 80 columns:
~~~c
#if defined(blabla) && \
  defined(another)